src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolver.php

```diff
@@ -47,7 +47,7 @@ public function resolve(Request $request, ArgumentMetadata $argument): array
         $options = $argument->getAttributes(MapEntity::class, ArgumentMetadata::IS_INSTANCEOF);
         $options = ($options[0] ?? $this->defaults)->withDefaults($this->defaults, $argument->getType());
 
-        if (!$options->class || $options->disabled) {
+        if (!$options->class) {
```

Comment:
This looses the ability to disable it by default though `$this->defaults`

Reply:
Good catch, wasn’t aware of that!